gateway/1.9/modules/ROOT/pages/flex-gateway-managed-upgrade-delete.adoc

```diff
@@ -0,0 +1,49 @@
+= Upgrade or Delete Managed Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+You can upgrade, or delete a Managed Flex Gateway using Anypoint Runtime Manager.
+
+Managed Flex Gateways are regularly auto-upgraded, depending on your Gateway release channel (LTS or Edge). 
+See xref:flex-gateway-version-lifecycle.adoc[] for more information.
+
+You can upgrade a Managed Flex gateway earlier than the regularly scheduled upgrade or patching process, between the new release date and the auto-upgrade date.
+You can choose the time that better suits the needs of your organization.
+
+== Early Upgrade Managed Flex Gateway (Optional)
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateways* in the navigation menu.
+. Select the *Managed Flex Gateway* tab.
+. Click the name of the gateway that you want to upgrade.
+. Click *Settings* in the navigation menu.
+. If a newer version is available, the *Version* field shows an *Update available* tag.
+. Click the padlock symbol next to the version and *Update available* tag.
+. Click the *Apply Changes* button. +
+A status message (*Your gateway is being updated*) will appear while the upgrade is being applied, and a success message ( *Your gateway has been successfully updated*) appears when the upgrade is complete.
+. Click *Dashboard* in the navigation menu to view your gateway status.
+
+== Delete Managed Flex Gateway
+
+You can delete a Managed Flex Gateway using Anypoint Runtime Manager.
+
+=== Before You Begin
+
+If there are any APIs associated with the gateway you want to delete, you must either move the APIs to another gateway or delete the APIs. After any associated APIs are moved or deleted, follow these steps to delete a gateway and confirm the deletion.
+
+=== Delete Gateway
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateways* in the navigation menu.
+. Select the *Managed Flex Gateway* tab.
+. Click the name of the gateway that you want to delete.
+. If the action menu button title is *Stop Gateway*, click the down arrow next to action menu button and select *Delete Gateway*.
+. Click the *Delete gateway* action menu button.
```

Comment:
```suggestion
. Click *Delete gateway*.
```

Reply:
Globally:
If a button is visible on the screen, there's no need to explain that it's an action menu button.
If a button is not visible, add the necessary steps to make it visible.